src/checks/y_check_comment_usage.clas.abap

```diff
@@ -70,19 +77,25 @@ CLASS y_check_comment_usage IMPLEMENTATION.
     LOOP AT ref_scan_manager->tokens ASSIGNING FIELD-SYMBOL(<token>)
     FROM statement-from TO statement-to
     WHERE type = scan_token_type-comment.
-      IF strlen( <token>-str ) >= 2 AND NOT
-         ( <token>-str+0(2) = |*"| OR
-           <token>-str+0(2) = |"!| OR
-           <token>-str+0(2) = |##| OR
-           <token>-str+0(2) = |*?| OR
-           <token>-str+0(2) = |"?| OR
-           ( strlen( <token>-str ) >= 3 AND <token>-str+0(3) = |"#E| ) OR
-           <token>-str CP '"' && object_name && '*.' ).   "#EC CI_MAGIC
+      IF strlen( <token>-str ) >= minimal_comment_len AND NOT is_comment_excluded( <token>-str ).
         comment_number = comment_number + 1.
       ENDIF.
     ENDLOOP.
   ENDMETHOD.
 
+  METHOD is_comment_excluded.
+
+      DATA(token_len) = strlen( token ).
+      result = xsdbool( token+0(minimal_comment_len) = |*"| OR
+                        token+0(minimal_comment_len) = |"!| OR
+                        token+0(minimal_comment_len) = |##| OR
+                        token+0(minimal_comment_len) = |*?| OR
+                        token+0(minimal_comment_len) = |"?| OR
+                        token CO |*| OR
+                        token_len >= exception_comment_len AND token+0(exception_comment_len) = |"#E| OR
```

Comment:
The parenthesis is needed here. Isn't it?

Reply:
if you like the magic numbers we can remove the anti-magic version